conf/defaults.ini

```diff
@@ -221,6 +221,9 @@ google_analytics_ua_id =
 # Google Analytics 4 tracking code, only enabled if you specify an id here
 google_analytics_4_id = 
 
+# Google Analytics 4 enhanced measurement is active on the client side
+google_analytics_4_enhanced_measurement_enabled = true
```

Comment:
this option name feels wrong or wierd to me, because setting this to true doesn't actually enable enhanced measurement, but it reflects the state of your "remote" google analytics account.

Maybe something like `send_manual_page_views`, with proper docs about why you would change/set this might be more suited? Thoughts?

Reply:
Yeah, I was also very back and forth on how to name it properly as this is something that has to be toggled in the google analytics account and not in Grafana. I wanted to make it generic so we can theoretically react in more areas to this enhanced measurement setting, but maybe YAGNI and renaming it to `google_analytics_4_send_manual_page_views` for now is better suited. 

I'll update the PR tomorrow.

Reply:
@joshhunt https://github.com/grafana/grafana/pull/57619/commits/fb9f35346a2ad657c976b6bfeedb2d9777e5bdb3